dev-manifest.yml

```diff
@@ -1,6 +1,7 @@
 memory: 256M
 applications:
   - name: verify-service-provider-dev
+    buildpack: java_buildpack
```

Comment:
This is a Cloud Foundary magic string. Run `cf buildpacks` to see which ones you can use.